### PR TITLE
[uss_qualifier] Use FlightInfoTemplate for flight intent storage

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
This PR continues the transition away from the legacy scd flight planning API by changing the flight intent storage format from being based on InjectFlightRequest (scd API) to being based on FlightInfoTemplate.  The CHE flight intent test data is updated to this new format, and the vertices are shortened to just a bounding box (rather than rounded-edge rectangles) for easier readability.  The Kentland flight data was unused, so therefore it is deleted in this PR to reduce maintenance burden.

To allow reuse, resolving a FlightIntentCollection (with delta definitions) to a concrete collection is moved to the FlightIntentCollection object itself.

Because verifying the new flight intent data was hard, a small script to render a FlightIntentCollection as a Google Earth KML is added to the test_data folder.

The wrong Polygon type was being referenced in geotemporal, so this is fixed.

### CODE REVIEW

File: monitoring/monitorlib/clients/flight_planning/flight_info_template.py [hunk 1]
```diff
@@ -51,3 +52,58 @@ def resolve(self, start_of_test: datetime) -> FlightInfo:
...
+                state = scd_api.OperationalIntentState.Nonconforming
+                off_nominal_volumes = volumes
+                volumes = []
+            elif info.basic_information.uas_state == UasState.Contingent:
+                state = scd_api.OperationalIntentState.Contingent
+                off_nominal_volumes = volumes
+                volumes = []
+            else:
+                raise ValueError(
+                    f"Unrecongized uas_state '{info.basic_information.uas_state}'"
```
Comment (mickmis) [inline monitoring/monitorlib/clients/flight_planning/flight_info_template.py:91]:
```suggestion
                    f"Unrecognized uas_state '{info.basic_information.uas_state}'"
```

File: monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/conflict_equal_priority_not_permitted/conflict_equal_priority_not_permitted.md [hunk 2]
```diff
@@ -76,9 +76,8 @@ Otherwise, the FlightIntentsResource must provide the following flight intents:
 </table>
 
 Because the scenario involves activation of intents, all activated intents must be active during the execution of the
-test scenario, i.e. they must start before the reference time plus planning time duration. Additionally, their end time
-must leave sufficient time for the execution of the test scenario. For the sake of simplicity, it is recommended to set
-the start and end times of all the intents to the same range.
+test scenario. Additionally, their end time must leave sufficient time for the execution of the test scenario. For the
+sake of simplicity, it is recommended to set the start and end times of all the intents to the same range.
```
Comment (mickmis) [inline monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/conflict_equal_priority_not_permitted/conflict_equal_priority_not_permitted.md:80]: This change should probably be also done on:
- monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/conflict_higher_priority/conflict_higher_priority.md
- monitoring/uss_qualifier/scenarios/astm/utm/flight_intent_validation/flight_intent_validation.md

File: monitoring/monitorlib/clients/flight_planning/flight_info_template.py (hunk 1 above)
Comment (BenjaminPelletier) [reply monitoring/monitorlib/clients/flight_planning/flight_info_template.py]: Thanks; done

File: monitoring/uss_qualifier/resources/overrides.py [hunk 3]
```diff
@@ -37,9 +37,21 @@ def _apply_overrides(base_object, overrides):
         return result_list
 
     elif isinstance(overrides, dict):
-        result = ImplicitDict.parse(base_object, type(base_object))
+        if isinstance(base_object, dict):
+            result = {k: v for k, v in base_object.items()}
+        elif base_object is None:
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/resources/overrides.py]: I agree -- I think the issue I was trying to address was separate from this so I've removed the clause

File: monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/conflict_equal_priority_not_permitted/conflict_equal_priority_not_permitted.md (hunk 2 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/conflict_equal_priority_not_permitted/conflict_equal_priority_not_permitted.md:80]: Sounds good -- done

**Pull request merged**